### PR TITLE
chore: Documentation on how the CLI works

### LINKED ISSUE #445: document how to use orb
so orb is a command line tool, but it turns out that you need to really understand noosphere's internals to make it work. When this issue is done, we should have documentation that shows how to create an identity, join a sphere, publish notes and read notes from others.

### PR DESCRIPTION
**Pull request by justinabrahms**
Fixes #445

### CODE REVIEW

**Comment by cdata** [comment]
LGTM, but need PR title to pass lint :)

File: rust/noosphere-cli/README.md [hunk 1]
```diff
@@ -5,3 +5,56 @@
...
+
+# ..now make edits ..
+
+# see the status of files in your directory
+orb status
+
+# persist changes to the sphere
+orb save
+
+# join a gateway, after you've set this identity as the counterpart
```
Comment (jsantell) [inline rust/noosphere-cli/README.md:31]: "join" means something else in this context e.g. in `orb sphere join`

Comment (jsantell) [inline rust/noosphere-cli/README.md:31]: Wonder if we can just get the counterpart sphere from the gateway after setting, though makes things a bit magical. In lieu of that, maybe just have a line for that like `orb config set counterpart <counterpart_did>`

Comment (justinabrahms) [reply rust/noosphere-cli/README.md:31]: How do you mean? From the linked video, this is what was typed to join the gateway.

Comment (cdata) [reply rust/noosphere-cli/README.md:31]: Yah, this is room for refinement. Today it doesn't work that way, so the docs are accurate.

Comment (jsantell) [reply rust/noosphere-cli/README.md:31]: i just mean the comment "join a gateway" in a pedantic-sense that maybe there's a better verb here (e.g. "register|subscribe to|set a gateway"), as we use `orb sphere join` later where "join" means synchronize another sphere (which is kinda what's going on here, admittedly)

File: rust/noosphere-cli/README.md [hunk 2]
```diff
@@ -5,3 +5,56 @@
...
+orb config set gateway-url <gatewayurl>
+
+# sync your changes with the upstream gateway
+orb sync
+
+# sync data from a different sphere. Note, you'll need to
+# `orb auth add <did>` from that other sphere.
+orb sphere join <their DID> --local-key `whoami`
```
Comment (cdata) [inline rust/noosphere-cli/README.md:39]: Sorry, just noticed this but: I think you need to join first, and then sync (currently steps as documented are in reverse order).

Comment (justinabrahms) [reply rust/noosphere-cli/README.md:39]:
```suggestion
orb sphere join <their DID> --local-key `whoami`
orb sync # pull in their changes
```
It was intended as a different sync.

File: rust/noosphere-cli/README.md [hunk 3]
```diff
@@ -5,3 +5,57 @@
...
+
+# ..now make edits ..
+
+# see the status of files in your directory
+orb status
+
+# persist changes to the sphere
+orb save
+
+# join a gateway, after you've set this identity as the counterpart
```
Comment (justinabrahms) [inline rust/noosphere-cli/README.md:31]:
```suggestion
# link your sphere to a gateway by setting this identity as the counterpart
```

**Review by cdata — APPROVED** [review]
Thanks for documenting this!

**Pull request merged**